packages/components/Molecules/Form/__tests__/RplSelect.test.js

```diff
@@ -29,8 +29,11 @@ describe('RplSelect', () => {
     ]
   }
 
+  const div = document.createElement('div')
+  document.body.appendChild(div)
+
   const baseConfig = {
-    attachToDocument: true,
+    attachTo: div,
```

Comment:
Replaced to attachTo as attachToDocument is getting deprecated in the coming version.